ibis/expr/types/relations.py

```diff
@@ -368,6 +431,12 @@ def __interactive_rich_console__(self, console, options):
         table = to_rich_table(self, width)
         return console.render(table, options=options)
 
+    def column(self, name: str | int) -> ir.Column:
```

Comment:
Is this a new API? If so, it should go into a separate PR.

Try to avoid introducing any new public APIs in this PR, it makes it much harder to review.

Since you're using it below, make it private for now to speed up the review.